compiler/test/formatter_outputs/aliases.gr

```diff
@@ -6,32 +6,32 @@ type Foo<a> = String
 
 type Foo<a, b> = String
 
-type Foo<a, b> = String // Test comment
+type Foo1<a, b> = String // Test comment
 
-type Foo< // Test comment
+type Foo2< // Test comment
   a,
-  b
+  b,
```

Comment:
I might not have understood what was what, but this shouldn't be trailing.

Reply:
Reverted